config/config.h.in

```diff
@@ -57,6 +57,8 @@
 #cmakedefine USE_FNAME_CASE
 #define USEMAN_S 1
 
+#cmakedefine USE_CRNL
```

Comment:
Oh wait, I should have read the cmake docs. It is dependent on the value from CMakeLists.